app/src/main/java/org/oppia/app/customview/interaction/FractionInputInteractionView.kt

```diff
@@ -0,0 +1,36 @@
+package org.oppia.app.customview.interaction
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.EditText
+import org.oppia.app.model.Fraction
+import org.oppia.app.model.InteractionObject
+
+// TODO(#249): These are the attributes which should be defined in XML, that are required for this interaction view to work correctly
+//  digits="0123456789/-"
+//  hint="Write fraction here."
+//  inputType="text"
+//  background="@drawable/edit_text_background"
+//  maxLength="200".
+
+/** The custom EditText class for fraction input interaction view. */
+class FractionInputInteractionView @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyle: Int = android.R.attr.editTextStyle
+) : EditText(context, attrs, defStyle), InteractionAnswerRetriever {
+
+  override fun getPendingAnswer(): InteractionObject {
+    val interactionObjectBuilder = InteractionObject.newBuilder()
+    if (!text.isNullOrEmpty()) {
+      interactionObjectBuilder.setFraction(
```

Comment:
Are we also handling mixed numbers, e.g.:
- ``23 1/2``
- ``-17 9/7``

Both of those are valid fraction answers.

Reply:
If we want to accept these inputs then we need to check user is entering input in proper format , with proper spacing.

Reply:
If user enter an input not as per this spacing and format then how we are supposed to handle it. We need to write some business logics for that it seems

Reply:
resolved